src/main/scala/org/hammerlab/guacamole/likelihood/Likelihood.scala

```diff
@@ -157,7 +157,7 @@ object Likelihood {
             case (allele, alleleFraction) =>
               alleleElementProbabilities(alleleToIndex(allele), ::) * alleleFraction
           }
-          sum( log( sum(alleleRows) )) + math.log(prior(genotype)) * depth
+          sum( log( sum(alleleRows) + 1e-10 )) + math.log(prior(genotype))
```

Comment:
IIRC, if `sum(alleleRows) == 0` (possible if you are using the alignment likelihood and all are empty) then `log` gives a NaN that propagates.